azerothcore/azerothcore-wotlk
# fix(DB/Creature): [Burning Steppes] Thauris Balgarr (rare) has no "Shoot" ability

[Burning Steppes] Thauris Balgarr (rare) has no "Shoot" ability #6979

- Closes #6979
- https://github.com/chromiecraft/chromiecraft/issues/1218


## Issues Addressed:

- Closes  https://github.com/azerothcore/azerothcore-wotlk/issues/6979

## SOURCE:


![image](https://user-images.githubusercontent.com/87535580/126000774-ff7fc819-c58b-4250-aa7e-8ce69f5b76d3.png)


## Tests Performed:

-  I attacked him while keeping the distance to see that he shoot me normally while using the net and the mine.

## How to Test the Changes:


1. Use the .go c id  8978, if you cant find it use .npc add temp 8978
2. Keeping the distance attack it
3. He should shoot you, use the net and the mine


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
